src/sentry/integrations/bitbucket/integration.py

```diff
@@ -96,8 +96,8 @@ def get_repositories(self, query=None):
                 for repo in resp.get("values", [])
             ]
 
-        exact_query = ('name="%s"' % (query)).encode("utf-8")
-        fuzzy_query = ('name~"%s"' % (query)).encode("utf-8")
+        exact_query = f'name="{query}"'.encode()
+        fuzzy_query = f'name~"{query}"'.encode()
```

Comment:
Do you not need to specify utf-8?

Reply:
I guess the default w/ no parameters is utf-8

Reply:
Oh I didn't even notice this. One of the commit hooks removed the parameter.